src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertifiedPerson/HealthCertifiedPersonViewController.swift

```diff
@@ -356,6 +361,10 @@ class HealthCertifiedPersonViewController: UIViewController, UITableViewDataSour
 			HealthCertificateCell.self,
 			forCellReuseIdentifier: HealthCertificateCell.reuseIdentifier
 		)
+		tableView.register(
+			MaskStateTableViewCell.self,
+			forCellReuseIdentifier: MaskStateTableViewCell.reuseIdentifier
+		)
```

Comment:
I think the add/remove logic is missing in `func tableView(_ tableView: UITableView, commit editingStyle: UITableViewCell.EditingStyle, forRowAt indexPath: IndexPath)`